pkg/apis/wavefront/v1alpha1/wavefrontcollector_types.go

```diff
@@ -17,6 +17,10 @@ type WavefrontCollectorSpec struct {
 	// Whether to deploy the collector as a daemonset. False will roll out as a deployment.
 	Daemon bool `json:"daemon,omitempty"`
 
+	// This flag is specifically for Openshift, if set to true, Collector will use default config bundled in the image
```

Comment:
nit: Lets move all openshift related spec to the bottom to better distinguish it.

Reply:
Done